feat: clearer error message for with-standard-malloc

HastD opened the pull request:
`ujust with-standard-malloc` requires unconfined-domain user namespace creation to be enabled. This detects when it's disabled and gives a more informative error message that tells the user how to fix it.

RoyalOughtness commented on files/justfiles/utilities.just:
```diff
@@ -172,7 +172,19 @@ toolbox-assemble CONTAINER="prompt" ACTION="create" FILE="/etc/toolbox/toolbox.i
 with-standard-malloc APP:
     #!/usr/bin/bash
-    bwrap --dev-bind / / --ro-bind /dev/null /etc/ld.so.preload "{{ APP }}"
+    source /usr/lib/ujust/libformatting.sh
```
why is this getting re-sourced? It's sourced globally for just already

HastD replied:
Is it? If I delete that line and test it with `just -f files/justfiles/utilities.just with-standard-malloc foo` (with unconfined userns creation disabled so the error path is followed), the final line isn't bolded, which suggests the variables from `libformatting.sh` haven't been set.

HastD replied:
Anyway, on second thought, I've removed the bold formatting—since `just` automatically prints out a bolded error message right afterward (because of the nonzero exit code), the bold formatting on that line looks a little weird.

RoyalOughtness replied:
it gets loaded for `ujust`, not `just`

HastD replied:
I see, that explains why it didn't work when I was testing it (using `just`) but does work "in production", so to speak.

HastD committed:
feat: clearer error message for with-standard-malloc

`ujust with-standard-malloc` requires unconfined-domain user namespace
creation to be enabled. This detects when it's disabled and gives a more
informative error message that tells the user how to fix it.

Signed-off-by: Daniel Hast <hast.daniel@protonmail.com>

HastD committed:
Merge branch 'live' into with-standard-malloc

HastD committed:
Merge branch 'live' into with-standard-malloc

HastD committed:
Merge branch 'live' into with-standard-malloc

RoyalOughtness commented on files/justfiles/utilities.just:
```diff
@@ -172,7 +172,18 @@ toolbox-assemble CONTAINER="prompt" ACTION="create" FILE="/etc/toolbox/toolbox.i
+    To allow this anyway, run:
+        ujust toggle-unconfined-domain-userns-creation
+    EOF
+        exit 1
```
from @RKNF404, worth looking into --unshare-userns-try first?

RKNF404 replied:
Doesn't work, mis-interpreted the use-case of that option
